ivy.xml

```diff
@@ -0,0 +1,98 @@
+        <dependency org="com.sun.mail" name="mailapi" rev="1.6.2"/>
+        <dependency org="com.jcraft" name="jsch" rev="0.1.55"/>
+        <dependency org="org.aspectj" name="aspectjrt" rev="1.8.10"/>
+        <dependency org="org.json" name="json" rev="20200518"/>
+
+        <dependency org="mil.army.usace.hec" name="cwms-db-jooq" rev="9.3.5">
+            <exclude org="org.jooq.pro" module="jooq-meta"/>
+            <exclude org="org.jooq.pro" module="jooq-codegen"/>
+        </dependency>
+        <dependency org="mil.army.usace.hec" name="cwms-db-aspects" rev="9.3.5"/>
+        <dependency org="mil.army.usace.hec" name="cwms-db-dao" rev="9.3.5"/>
+
+        <dependency org="mil.army.usace.hec" name="hec-monolith" rev="2.0.2">
+            <exclude org="codebase"/>
+            <exclude org="org.jdom"/>
+            <exclude org="org.bouncycastle"/>
+            <exclude org="org.eclipse.jetty.aggregate"/>
+            <exclude org="javax.servlet"/>
+            <exclude org="mil.army.usace.hec.swingx"/>
+            <exclude org="com.fifesoft"/>
+            <exclude org="org.python"/>
+            <exclude org="com.vividsolutions"/>
+            <exclude org="mil.army.usace.hec" module="context-specific-help"/>
+            <exclude org="mil.army.usace.hec" module="hec-geojson"/>
+            <exclude org="javax.mail"/>
+            <exclude org="org.apache.poi"/>
+            <exclude org="net.java.dev.jna"/>
+            <exclude org="org.apache-commons" module="common-vfs2"/>
+            <exclude org="org.jfree"/>
+            <exclude org="com.mrsid"/>
+            <exclude org="javax.help"/>
+        </dependency>
+        <dependency org="mil.army.usace.hec" name="hec-cwms-ratings-core" rev="1.1.0"/>
+
+
+        <!-- test dependencies -->
+        
+        <dependency org="org.junit.platform" name="junit-platform-launcher" rev="1.8.2" conf="test_platform->default"/>
+        <dependency org="org.junit.platform" name="junit-platform-engine" rev="1.8.2" conf="test_platform->default"/>
+        <dependency org="org.junit.platform" name="junit-platform-commons" rev="1.8.2" conf="test_platform->default"/>
+        <dependency org="org.junit.jupiter" name="junit-jupiter-api" rev="5.8.1" conf="test->default"/>
+        <dependency org="org.junit.jupiter" name="junit-jupiter-engine" rev="5.8.1" conf="test->default"/>
+
+        <!-- izpack -->
+        <dependency org="org.codehaus.izpack" name="izpack-standalone-compiler" rev="4.3.5" conf="izpack->default"/>
+
+        <!-- groovy deps for biuld script -->
+        <dependency org="org.codehaus.groovy" name="groovy-ant" rev="3.0.10" conf="groovy->default"/>
+        <dependency org="org.codehaus.groovy" name="groovy" rev="3.0.10" conf="groovy->default"/>
+        <dependency org="org.codehaus.groovy" name="groovy-xml" rev="3.0.10" conf="groovy->default"/>
+
+        <!-- keep unused transient dependencies out -->
+
+        <exclude org="mil.army.usace.hec" module="loading-gui-lib"/>
```

Comment:
These `mil.army.usace.hec` dependency excludes can be removed when an hec-monolith 3.0.0 is released (non-rc).